### DISCUSSION TITLE
For SF6: Is there anyway to be able to move the free cam while the pause menu is up?

### DISCUSSION THREAD

**Discussion by Cyberpunk102** [praydog/REFramework#830] [General Questions]
I was hoping to get some cool shots during matches, but I can't really get any cool poses of moves being done because I am unable to move the camera when the game is paused. Please forgive my ignorance - I'm used to using Nvidia Ansel and being able to freeze a match and move around whenever. So I found RE Framework as an alternative when I realized SF6 didn't support Ansel. I thought using the pause function during replays might still let me move around, but it still basically disables the camera well. Any advice from people more familiar with RE Framework would be much appreciated!

**Comment by praydog** [comment]
I recommend you head over to the [Modding Haven](https://discord.gg/9Vr2SJ3) discord and look for alphaZomega's Lua Freecam script for REFramework. It has much more functionality than REF's built in freecam and especially for SF6 it has some nice features.
